cloc randomly failing on travis

Failed: https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/341726531?utm_source=github_status&utm_medium=notification

Same code passed: https://travis-ci.org/AnalyticalGraphicsInc/cesium/builds/341726533?utm_source=github_status&utm_medium=notification

Failed output
```
6.25s$ npm run cloc
> cesium@1.42.1-primitive-image-resource25359 cloc /home/travis/build/AnalyticalGraphicsInc/cesium
> gulp cloc
[03:37:51] Using gulpfile ~/build/AnalyticalGraphicsInc/cesium/gulpfile.js
[03:37:51] Starting 'clean'...
[03:37:51] Finished 'clean' after 343 ms
[03:37:51] Starting 'cloc'...
Unable to read copyrightHeader.js; ignoring.
Can't create unknown regex: $RE{comment}{C++} at node_modules/cloc/lib/cloc line 9619.
	...propagated at node_modules/cloc/lib/cloc line 4789.
[03:37:54] 'cloc' errored after 2.74 s
[03:37:54] Error: Command failed: perl node_modules/cloc/lib/cloc --quiet --progress-rate=0 Source/ --exclude-dir=Assets,ThirdParty --exclude-list-file=copyrightHeader.js
Unable to read copyrightHeader.js; ignoring.
Can't create unknown regex: $RE{comment}{C++} at node_modules/cloc/lib/cloc line 9619.
	...propagated at node_modules/cloc/lib/cloc line 4789.
    at ChildProcess.exithandler (child_process.js:275:12)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cesium@1.42.1-primitive-image-resource25359 cloc: `gulp cloc`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cesium@1.42.1-primitive-image-resource25359 cloc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-02-15T03_37_55_635Z-debug.log
The command "npm run cloc" exited with 1.
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -144,7 +144,7 @@ gulp.task('cloc', ['clean'], function() {
     //Run cloc on primary Source files only
     var source = new Promise(function(resolve, reject) {
         cmdLine = 'perl ' + clocPath + ' --quiet --progress-rate=0' +
-                  ' Source/ --exclude-dir=Assets,ThirdParty --exclude-list-file=copyrightHeader.js';
+                  ' Source/ --exclude-dir=Assets,ThirdParty --not-match-f=copyrightHeader.js';
 
         child_process.exec(cmdLine, function(error, stdout, stderr) {
             if (error) {

